app/src/main/java/com/rentminder/Icon&Text.kt

```diff
@@ -1,106 +1,79 @@
 package com.rentminder
 
-import androidx.compose.foundation.layout.Arrangement
 import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.layout.size
 import androidx.compose.material.Icon
 import androidx.compose.material.Text
-import androidx.compose.material.icons.Icons
-import androidx.compose.material.icons.outlined.Home
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import androidx.compose.ui.res.painterResource
-import androidx.compose.ui.text.TextStyle
-import androidx.compose.ui.text.style.TextDecoration
 
 @Composable
-fun RentIconText() {
-    Row(verticalAlignment = Alignment.CenterVertically){
-        Icon(painterResource(
-            id = R.drawable.outline_house_24
-        ),
-            contentDescription = "Rent Icon",
-            modifier = Modifier
-                .size(30.dp)
+fun IconText(
+    iconResId: Int,
+    contentDescription: String,
+    text: String
+){
+    Row(verticalAlignment = Alignment.CenterVertically) {
+        Icon(
+            painter = painterResource(id = iconResId),
+            contentDescription = contentDescription,
+            modifier = Modifier.size(30.dp)
         )
         Text(
-            text = "Rent",
-            fontSize = (26.sp)
+            text = text,
+            fontSize = 26.sp
```

Comment:
Created a function that allows for icons to be made in order to eliminate repeating code in the creation of multiple icons. This leads to less code, better maintainability, and makes it easier to update in the future if needed.

https://scottc130.medium.com/using-functions-to-avoid-code-repetition-1fe4beff6d4d
https://developer.android.com/codelabs/basic-android-kotlin-compose-functions#7
https://kt.academy/article/kfde-functions

Reply:
Hmmm, neat.  Good idea.

While you're here, externalize strings to strings.xml as well.